tests/service-catalog/Dockerfile

```diff
@@ -15,11 +15,13 @@ RUN CGO_ENABLED=0 go test -tags=acceptance -c ./test -o /servicecatalog.test
 FROM alpine:3.10
 RUN apk add --no-cache curl
 
+WORKDIR /app
+
 COPY --from=builder env-tester.bin /go/bin/env-tester.bin
 COPY --from=builder ./servicecatalog.test .
 COPY ./licenses/ /app/licenses
 COPY --from=builder /entrypoint.sh .
 
 LABEL source=git@github.com:kyma-project/kyma.git
 
-ENTRYPOINT ./entrypoint.sh
+ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
I guess at least this should be now `["/entrypoint.sh"]`, because of the `WORKDIR /app` on line 18.

Reply:
it's already copied to the local directory. See line 23

Reply:
ah, my bad